config/environments/test.rb

```diff
@@ -3,6 +3,8 @@
 # your test database is "scratch space" for the test suite and is wiped
 # and recreated between test runs. Don't rely on the data there!
 
+require 'dotenv/load'
```

Comment:
We should be able to get rid of this, too.

Reply:
Ah gotcha - I didn't realise you meant both lines! Done :)